dev/versions.md

```diff
@@ -64,7 +62,7 @@ _note: docker client installation may be able to be improved, see https://github
 
 ### cortexlabs/yaml
 
-1. Check [go-yaml/yaml](https://github.com/go-yaml/yaml) to see if there were new releases since [cortexlabs/yaml](https://github.com/cortexlabs/yaml)
+1. Check [go-yaml/yaml](https://github.com/go-yaml/yaml/commits/v2) to see if there were new releases since [cortexlabs/yaml](https://github.com/cortexlabs/yaml/commits/v2)
```

Comment:
That is a good point, now that you mention it, I would have expected it to work like you wrote. It does seem to work like it is, so I think I'll leave it as is for now. It works as is probably because we specify the commit SHA to use (via `go get github.com/cortexlabs/yaml@581aea36a2e4db10f8696587e48cac5248d64f4d`), and it locates it in the v2 branch.

Reply:
That said, I can try changing it if you feel that it would be better, lmk what you think

Reply:
Okay, got it. I'm good with leaving it as it is then.